### PR TITLE
landing-page

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please review it_**
hompe-page
Landing-page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+body {
+    text-align: center;
+}
+
+.header-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 3rem;
+    background-color: #fff1f2;
```
Comment (NeerajVodala) [inline index.css:10]: can use a variable here

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

**Review by nightlight-wander — APPROVED** [review]
looks good to me

File: index.css (hunk 1 above)
Comment (anshtripathi01) [reply index.css:10]: i've used only here.

**Pull request merged**